### DISCUSSION TITLE
Alternative for Adding basePath to image src

### DISCUSSION THREAD

**Discussion by MrMirfan** [gregrickaby/nextjs-github-pages#60] [General]
Instead of Adding BasePath like this

`src="/nextjs-github-pages/vercel.svg"`

We can prepend the src with . and it works on Both Local and Prod

`src="./profile/passport.avif"`

**Comment by realChakrawarti** [comment]
@MrMirfan You mean here https://github.com/gregrickaby/nextjs-github-pages/blob/main/app/page.tsx#L20 ? So, instead of `src="/nextjs-github-pages/vercel.svg"`, we could write `src="./vercel.svg"` ?

I am facing an issue where images public images doesn't load, https://realchakrawarti.github.io/aham/. Link to the repo: https://github.com/realChakrawarti/aham

**Reply by MrMirfan** [reply]
Oh sorry I thought it was a fix though it worked for me.

I'll sure look into it and update here if there's any fix

**Comment by truongngoctuan** [comment]
The solution with "./" works. I guess "/" will start from root, while "./" will start from your basePath.

**Reply by MrMirfan** [reply]
Thanks for confirming
